app/Controllers/feedController.php

```diff
@@ -480,8 +481,11 @@ public static function actualizeFeed($feed_id, $feed_url, $force, $simplePiePush
 							$entryDAO->beginTransaction();
 						}
 						$entryDAO->addEntry($entry->toArray());
-						$feed->incPendingUnread();
-						$nb_new_articles++;
+
+						if (!$entry->isRead()) {
+							$feed->incPendingUnread();
+							$nb_new_articles++;
+						}
```

Comment:
```suggestion
						}
						$nb_new_articles++;
```

I believe `$nb_new_articles++` needs to be out of the `if` condition, as we want to increase it for all new articles, even when new articles are automatically marked as read for one reason or another